thanos/charts/values.yaml

```
@@ -282,6 +288,9 @@ thanos:
     # -- Labels to add to the Thanos Compactor service
     serviceLabels: {}
 
+    # -- Annotations to add to the Thanos Compactor service
```

```suggestion
    # -- Service specific annotations to add to the Thanos Compactor service in addition to its already configured annotations.
```

The others similar please.